sceptre/cli.py

```diff
@@ -630,6 +630,11 @@ def init_project(ctx, project_name):
         folder_path = os.path.join(project_folder, folder)
         os.makedirs(folder_path)
 
+    # Create .gitignore file in project
+    gitignore_path = os.path.join(project_folder, ".gitignore")
+    with open(gitignore_path, "wt") as gitignore_file:
+        gitignore_file.write("*.pyc")
+
```

Comment:
I feel like your changes should be its own function instead of inlined.

Reply:
I wasn't sure about that, but it probably should. Maintaining it would be easier that way.